### ISSUE TITLE
Narrator is narrating incorrect role and not narrating state for dropdowns present in the 'SQL Pref Main' page, while navigating using focus/scan mode.

### ISSUE THREAD

**Issue by JamesFerebee** [microsoft/SqlNexus#340]
Other Instances:
1. Same issue is observed for the Data import Dialog.
Issue: Screen reader (Narrator/JAWS) are narrating incorrect role as Link for the 'Options' button and not narrating the State as expanded/collapsed, present in the Data import Dialog, while navigating using Scan/Scan off mode.
Observation: Narrator is narrating as 'options link'
Expected: Screen reader (Narrator/JAWS) should narrate correct role as button for the 'Options' button and state as expanded/collapsed, present in the Data import Dialog, while navigating using Scan/Scan off mode.
Ex: Narrator should announce as 'Options button Collapsed'.
path: SQLNexus -> connect server dialog->  
Connect to server Dialog gets displayed and invoke connect button-> Navigate to Import button in left navigation and invoke it-> Data Import Dialog gets displayed-> navigate to Options control present in the dialog.

Impact on a Person with Disability (PWD):
Users who rely on screen readers will not be aware of the control as Narrator is narrating incorrect role and not narrating state for dropdowns present in the 'SQL Pref Main' page, while navigating using focus/scan mode.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Screen readers: Narrator, JAWS (2022.2110.6).

Pre-Requisites:
1. Install SQL NEXUS application on the desktop.
2. Enable Narrator/JAWS

Repro Steps:
Step 1: Open the SQL Nexus app in desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: 'SQL Pref Main' page getting displayed->Navigate the dropdown controls using focus/scan mode.
Step 4: Verify whether Narrator is narrating incorrect role and not narrating state for dropdowns present in the 'SQL Pref Main' page, while navigating using focus/scan mode or not.
 

Actual Result:
Narrator is narrating incorrect role and not narrating state for dropdowns present in the 'SQL Pref Main' page, while navigating using focus/scan mode.
Observation: 
1.Reporter
2.Tasks
3.Logs
Narrator behavior: Narrator announces as 'Reporter link' when focus lands on reporter dropdown.

JAWS behavior: 

1.Same issue is observed for JAWS tab mode.

2.JAWS focus is not moving to the controls using cursor mode.

Expected Result:

Narrator Should narrate correct role and should narrate state for dropdowns present in the 'SQL Pref Main' page, while navigating using focus/scan mode.
Ex:
Narrator should announce as 'Reporter button collapsed/expanded' when focus lands on 'Reporter dropdown.

Suggestive Fix:

Use Aria attribute as aria-expanded="true".
Don't use scripting to transform static elements, such as a span or div to create interactive elements.
If you use scripting to create the static or interactive elements, use AIRA attributes based on component or element behavior.

**Comment by JamesFerebee** [comment]
1711617
